src/sonic-yang-models/yang-models/sonic-kubernetes_master.yang

```diff
@@ -0,0 +1,60 @@
+module sonic-kubernetes_master {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-kubernetes_master";
+    prefix kubernetes_master;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    description "KUBERNETES_MASTER YANG Module for SONiC OS";
+    
+    revision 2022-10-09 {
+        description "First Revision";
+    }
+
+    container sonic-kubernetes_master {
+
+        container KUBERNETES_MASTER {
+
+            description "KUBERNETES_MASTER part of config_db.json";
+
+            container SERVER {
+
+                leaf ip {
+                    description "Kubernetes api server endpoint";
+                    type string;
```

Comment:
This should use IPAddress type

Reply:
It should be IPAddress type and DNS domain type, currently use inet:host to cover both of them.